env/dev/env/android/main.cljs

```diff
@@ -1,10 +1,10 @@
 (ns ^:figwheel-no-load env.android.main
   (:require [reagent.core :as r]
-            [re-frame.core :as re-frame]
             [status-im.android.core :as core]
-            [figwheel.client :as figwheel :include-macros true]
+            [figwheel.client :as fw]
```

Comment:
We usually use full name as alias so `figwheel` was good here.

Reply:
Thanks for your feedback.
When I ran `re-natal upgrade`, it automatically made changes to this file.
Will fix.